lib/internal/test_runner/reporter/tap.js

```diff
@@ -46,8 +46,14 @@ async function * tapReporter(source) {
         yield `${indent(data.nesting)}# Subtest: ${tapEscape(data.name)}\n`;
         break;
       case 'test:stderr':
-      case 'test:stdout':
-      case 'test:diagnostic':
+      case 'test:stdout': {
+        const lines = StringPrototypeSplit(data.message, kLineBreakRegExp);
+        for (let i = 0; i < lines.length; i++) {
+          if (lines[i].length === 0) continue;
+          yield `${indent(data.nesting)}# ${tapEscape(lines[i])}${i < lines.length - 1 ? '\n' : ''}`;
```

Comment:
`stderr` and `stdout` doesn't have `nesting`.
Shall we remove `${indent(data.nesting)}`?